dolphinscheduler-server/src/test/java/org/apache/dolphinscheduler/server/master/DependentTaskTest.java

```diff
@@ -16,10 +16,18 @@
  */
 package org.apache.dolphinscheduler.server.master;
 
+import java.util.stream.*;
```

Comment:
Hi,
Please configure checkstyle.xml and code style xml in style directory. After configure that, you can use option+command+L to automatic format the code.

Checkstyle.xml configuration lesson: https://www.cnblogs.com/wanshi1989/p/5478050.html
code style xml configuration lesson: https://blog.csdn.net/qq_34579060/article/details/80100040

Reply:
Hi, my checkstyle install failed with:

```
com.puppycrawl.tools.checkstyle.api.CheckstyleException: cannot initialize module TreeWalker - TreeWalker is not allowed as a parent of LineLength Please review 'Parent Module' section for this Check in web documentation if Check is standard.
```

any idea of this?

Reply:
Solution refers to issue #3432 .